llvmlite/ir/builder.py

```diff
@@ -837,6 +842,7 @@ def cbranch(self, cond, truebr, falsebr):
         br = instructions.ConditionalBranch(self.block, "br",
                                             [cond, truebr, falsebr])
         self._set_terminator(br)
+        self.set_pred(truebr); self.set_pred(falsebr)
```

Comment:
Would look with the API change mentioned in the review as:

```suggestion
        truebr.add_predecessor(self)
        falsebr.add_predecessor(self)
```

Reply:
thank you for the note, the meaning of `set_pred` was to set self.basic_block as predecessor of the parameter, but it effectly adds to predecessors, then the problem was just the name of the function